packages/rdf-editor/src/RdfEditor.js

```diff
@@ -179,7 +183,7 @@ export class RdfEditor extends Editor {
     })
 
     if (!quadStream) {
-      this.serialized = `No serializer found for media type ${this.format}`
+      this.value = `No serializer found for media type ${this.format}`
```

Comment:
This has nothing to do with this feature right?

Reply:
no. missed that in #45